app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+    openRecommendedMediaItem: (MediaItem) -> Unit,
+    playTrailer: (String) -> Unit ,
+    modifier: Modifier = Modifier
+){
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .background(color = MaterialTheme.colorScheme.surfaceContainer)
+    ){
+        val columnState = rememberScrollState()
+        ShowScreenAppBar(
+            navigateBack = navigateBack,
+            addToFavorites = {
+                // TODO - cache MediaItem
+            },
+            modifier = Modifier
+                .offset(y = dimensionResource(R.dimen.app_bar_y_offset))
+                .padding(end = dimensionResource(R.dimen.show_screen_horizontal_padding))
+                .zIndex(1f)
+                .fillMaxWidth()
+        )
+        Column(
+            horizontalAlignment = Alignment.Start,
+            modifier = Modifier
+                .fillMaxSize()
+                .background(color = MaterialTheme.colorScheme.surfaceContainer)
+                .verticalScroll(
+                    state = columnState,
+                )
+        ) {
+            Box(
+                modifier = Modifier.fillMaxWidth(),
+                contentAlignment = Alignment.Center
+
+            ) {
+                ShowScreenCoverImage(
+                   imagePath = mediaItem.coverPath,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .aspectRatio(1f)
+                )
+                ShowScreenMediaBasicInfo(
+                    title = mockMovie.title,
+                    date = mockMovie.date,
+                    language = mockMovie.language,
+                    progress = mockMovie.rating,
+                    duration = getMinutesString(mockMovie.duration),
+                    seasons = null,
+                    modifier = Modifier
+                        .align(alignment = Alignment.BottomCenter)
+                        .fillMaxWidth()
+                        .padding(horizontal = dimensionResource(R.dimen.show_screen_horizontal_padding))
+
+                )
+            }
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_1))
+            )
+            HorizontalDivider(
+                thickness = dimensionResource(R.dimen.horizontal_divider_thickness),
+                color = MaterialTheme.colorScheme.scrim,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = dimensionResource(R.dimen.show_screen_horizontal_padding))
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_1))
+            )
+            Text(
+                text = mockMovie.overview,
+                style = MaterialTheme.typography.bodySmall.copy(
+                    fontWeight = FontWeight.Normal,
+                    fontSize = 12.sp,
+                    color = white_CC,
```

Comment:
What is white__CC color? Also, if you are adding multiple values in `copy` for a typography you can use one with more similar or add a new typography. `bodySmall` already has `FontWeight.Normal` and `16.dp` line height. `copy` won't change those values so you don't need to re-pass them

Reply:
I removed sufficient parts of the code. Thanks! I don't know why I wrote it like this.